include/pgduckdb/types/pgduckdb_enum.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "duckdb/common/types/vector.hpp"
+#include "duckdb/common/types.hpp"
+
+extern "C" {
+#include "postgres.h"
+#include "catalog/pg_enum.h"
+#include "catalog/pg_type.h"
+#include "utils/syscache.h"
+#include "access/htup_details.h"
+}
+
+namespace pgduckdb {
+
+using duckdb::idx_t;
+using duckdb::LogicalType;
+using duckdb::Vector;
+
+struct PGDuckDBEnum {
```

Comment:
Should this be changed to a namespace? Now that this struct doesn't have any fields anymore?

Reply:
Yea possibly, in our codebase we're not too strict on static-membered-structs vs namespaces, I think when we also need a constexpr value here we can't use a namespace but can use a struct, so I almost always go for using a struct